app/controllers/scans_controller.rb

```diff
@@ -27,20 +27,11 @@ def new_request_params
   end
 
   def redirect_to_success_with_token
-    redirect_to illiad_query(current_request)
+    redirect_to illiad_url
```

Comment:
Ideally it would be better to do it on the back-end, but ILLilad does not have a supported API for handling request creation. If we wanted to do it on the back-end, we would have to insert rows directly into the MSSQL tables without intimately knowing the schemas and relationships between tables. It might be worth looking into this angle, but it would take a lot of investigation and testing before we could go live.